core/src/main/java/org/geysermc/geyser/entity/type/Tickable.java

```diff
@@ -29,5 +29,17 @@
  * Implemented onto anything that should have code ran every Minecraft tick - 50 milliseconds.
  */
 public interface Tickable {
+    /*
```

Comment:
Make both of these Javadoc comments instead of just comments, please!

Reply:
Should be fixed in latest commit.